deploy/deployment.yaml

```diff
@@ -38,8 +38,16 @@ spec:
     spec:
       serviceAccountName: ekco
       restartPolicy: Always
-      nodeSelector:
-        node-role.kubernetes.io/master: ""
+      affinity: 
+        nodeAffinity: 
+          requiredDuringSchedulingIgnoredDuringExecution: 
```

Comment:
can you please test this in a cluster that only has one of two labels?

Reply:
tested this on real [cluster](34.66.76.113)